blazor/3D-chart/appearance.md

```diff
@@ -0,0 +1,559 @@
+		new Chart3DData{ Country="China", Gold=40 },
+		new Chart3DData{ Country= "Japan", Gold=70 },
+		new Chart3DData{ Country= "Australia", Gold=60},
+		new Chart3DData{ Country= "France", Gold=50 },
+		new Chart3DData{ Country= "Germany", Gold=40 },
+		new Chart3DData{ Country= "Italy", Gold=40 },
+		new Chart3DData{ Country= "Sweden", Gold=30 }
+    };
+}
+
+```
+
+![Blazor Column 3D Chart with Custom Background and Border](images/appearance/blazor-column-chart-custom-background-and-border.png)
+
+**Chart margin**
+
+The 3D chart's margin can be set from its container using the [Chart3DMargin](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartMargin.html) property.
+
+```cshtml
+
+@using Syncfusion.Blazor.Chart3D
+
+<SfChart3D Title="Olympic Medals" BackgroundColor="skyblue" WallColor="transparent" EnableRotation="true" RotationAngle="7" TiltAngle="10" Depth="100">
+    <Chart3DBorder Color="#FF0000" Width="2"></Chart3DBorder>
+    <Chart3DMargin Left="60" Right="60" Top="60" Bottom="60"></Chart3DMargin>
+
+    <Chart3DPrimaryXAxis ValueType="Syncfusion.Blazor.Charts.ValueType.Category" />
+
+    <Chart3DSeriesCollection>
+        <Chart3DSeries DataSource="@MedalDetails" XName="Country" YName="Gold" Type="Chart3DSeriesType.Column">
+        </Chart3DSeries>
+    </Chart3DSeriesCollection>
+</SfChart3D>
+
+@code{
+    public class Chart3DData
+    {
+        public string Country { get; set; }
+        public double Gold { get; set; }
+    }
+
+    public List<Chart3DData> MedalDetails = new List<Chart3DData>
+	{
+		new Chart3DData{ Country= "USA", Gold=50  },
+		new Chart3DData{ Country="China", Gold=40 },
+		new Chart3DData{ Country= "Japan", Gold=70 },
+		new Chart3DData{ Country= "Australia", Gold=60},
+		new Chart3DData{ Country= "France", Gold=50 },
+		new Chart3DData{ Country= "Germany", Gold=40 },
+		new Chart3DData{ Country= "Italy", Gold=40 },
+		new Chart3DData{ Country= "Sweden", Gold=30 }
+    };
+}
+
+```
+
+![Blazor Column 3D Chart with Custom Margin](images/appearance/blazor-column-chart-custom-margin.png)
+
+## Animation
+
+To customize the animation for a particular series, the [Animation](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.ChartSeries.html#Syncfusion_Blazor_Charts_ChartSeries_Animation) property can be used. It can be enabled or disabled by using the [Enable](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartCommonAnimation.html#Syncfusion_Blazor_Charts_StockChartCommonAnimation_Enable) property. The [Duration](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartCommonAnimation.html#Syncfusion_Blazor_Charts_StockChartCommonAnimation_Duration) property specifies the duration of an animation and the [Delay](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.StockChartCommonAnimation.html#Syncfusion_Blazor_Charts_StockChartCommonAnimation_Delay) property allows us to start the animation at desire time.
```

Comment:
@IlamathiSF3875 
Remove other chart API links and highlight the property. Check the Palettes added comment.